### ISSUE TITLE
Where is the code?

### ISSUE THREAD

**Issue by binflo13** [aaronnabil/halcon-calplate#2]
Were you able to finally develop this functionality? It would be very interesting.

**Comment by aaronnabil** [comment]
Yes it is finished, I didn't publish it because mvtec had some errata to the calibration plate process I didn't understand and it meant loading a bunch of versions of halcon to compare plates to make sure they weren't affected.

If it isn't clear from the notes it is very unlikely you need this repo, it simply allows you to recreate (and has pre-generated versions of) the exact calibration plates mvtec sells commercially.  If you actually have a halcon license you don't need this as halcon has the ability to generate (and use) any plate built in and you aren't limited to using the commercial plate sizes (nor is there any advantage to them, they are just arbitrary sizes).  So if you have halcon, just generate a plate yourself and use it, and if you don't have halcon, why would you need this script?

The pre-generated files in the repo would only be useful in the very rare situation that you only had a runtime version and some existing halcon script that someone wrote that relied on the exact commercially sold mvtec calibration plate and you needed to print another one.  Look like someone actually sells these (!) which is kind of weird so I guess publishing them would also make him not so happy.  https://www.hcv-store.com/calibration-plates-pdf-demo
